common-lib/src/main/scala/com/gu/mediaservice/lib/config/CommonConfig.scala

```diff
@@ -204,6 +204,7 @@ abstract class CommonConfig(resources: GridConfigResources) extends AwsClientBui
    * and the singleton instance added to the list.
    */
   val applicableUsageRights: Seq[UsageRightsSpec] = configuration.get[Seq[UsageRightsSpec]]("usageRights.applicable")
+  val stdUserExcludedUsageRights = getStringSet("usageRights.stdUserExcluded")
```

Comment:
This new config field can be safely ignored by the Guardian and others because `getStringSet` will default to empty if the config field is non present 👍